src/main/java/hudson/remoting/RemoteClassLoader.java

```diff
@@ -322,7 +323,6 @@ ClassReference toRef(ClassFile2 cf) {
                                     // but we need to remember to set the interrupt flag back on
                                     // before we leave this call.
                                     interrupted = true;
-                                    continue;   // JENKINS-19453: retry
```

Comment:
Isn't this a behaviour change?

Reply:
I don't think so. As IntelliJ says, "'continue' is unnecessary as the last statement in a loop ".

Because of the depth of the loops it's hard to read and the GH UI here doesn't make it any easier.

Reply:
Looks like a redundant `continue` as it's the end of the block.

Reply:
Makes sense, thanks for the clarification!